package.json

```diff
@@ -23,6 +23,7 @@
     "gulp-beautify": "~3.0.0",
     "gulp-filter": "~7.0.0",
     "node-forge": "~0.10.0",
+    "selfsigned": "^2.0.1",
```

Comment:
```suggestion
    "selfsigned": "~2.0.1",
```

Reply:
also make sure to run `standard --fix` before committing, not sure this will pass the linter

Reply:
don't forget to merge the suggestion i put above

some more things to do:

alter this:

```
To run the app:
- Install dependencies: npm i
- To run in dev mode:   npm run dev
- To run in prod mode:  npm run prod
Once running, visit http://localhost:10709
```

The link should include the HTTPS link, not the HTTP link.

also the HTTP server should be disabled. Make it just HTTPS.

and randomize the port of the HTTPS server so it's not always 43733. there is an example of how to do a random port in the HTTP code. just reuse that